Modules/SubModule_Population/xASL_stat_GetDICOMStatistics.m

```diff
@@ -154,11 +154,27 @@ function xASL_stat_GetDICOMStatistics(x, ScanType, HasSessions, bOverwrite)
 
 %% 3) Write TSV file
 
-% Remove empty elements
-TSV(cellfun('isempty',TSV)) = {'_'};
-
-% Remove nan elements
-TSV(cell2mat(cellfun(@(x)any(isnan(x)),TSV,'UniformOutput',false))) = {'n/a'};
+% Ensure all elements of the TSV cell array have the correct format
+for iRow=1:size(TSV,1)
+    for iColumn=1:size(TSV,2)
+        % Remove lists (we had a problem where it was tried to insert SliceTiming arrays into the table)
+        if size(TSV{iRow,iColumn},1)>1
+            TSV{iRow,iColumn} = 'n/a';
```

Comment:
But shouldn't this produce a warning? The pipeline should never suddenly introduce lists inside one cell. This could be the case in the import module, but not in the other modules.

Reply:
Sure, then we can keep track of the warning and limit the number of its outputs

Reply:
I added a warning for now, let's see how it turns out fd9f2f4

Reply:
I have an alternative solution for more transparency. If there's a list, then report it is a list instead of "n/a". You can either print "list" or "field1, field2,..."